A/Atomsk/build_tarballs.jl

```diff
@@ -7,25 +7,35 @@ version = v"0.11.2"
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/pierrehirel/atomsk.git", "3333858281e0ebd6279825b83ab871cf4d050a8d")
+    GitSource("https://github.com/pierrehirel/atomsk.git", "84f60a20c5b814fec03bffe1bccc3daaed0fc65d"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/atomsk/src/
-mkdir -p ${bindir}
+cd $WORKSPACE/srcdir/atomsk/src
+sed -i '172s/$/ -I $(OBJ) $(LAPACK)/' Makefile
+sed -i '172s/atomsk.so/\$(libdir)\/libatomsk.$(dlext)/' Makefile
+if [[ "${exext}" == "" ]]; then
+    sed -i '161s/\$(BIN)/\$(bindir)\/atomsk/' Makefile
+else
+    sed -i '161s/\$(BIN)/\$(bindir)\/atomsk.$(exeext)/' Makefile
+fi
```

Comment:
Can you please add a proper patch?  `sed` patching-on-the-fly is fragile and error-prone when changing version.

Reply:
I was having difficulty with that, but let me try again, and I'll see if I can get it to work. It was telling me that the patch failed.

Reply:
The last commit has the patch, but running `git diff -p` produces a broken patch. Do you know why this is?